admin/javascript/src/components/breadcrumb/index.js

```diff
@@ -24,7 +24,7 @@ class BreadcrumbComponent extends SilverStripeComponent {
      ]),
        this.props.crumbs.slice(-1).map((crumb, index) => [
          <li className="breadcrumb__item breadcrumb__item--last">
-           <h2 className="breadcrumb__item-title breadcrumb__item-title--last" key={index}>
+           <h2 className="breadcrumb__item-title breadcrumb__item-title--last text-truncate" key={index}>
```

Comment:
Yup sure, .truncate was created before hand but should be an easy fix.
The breadcrumb links when there are too many (top line) don't truncate at the moment and might be a bit fiddly to find a good solution.